server/model.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable linebreak-style */
```

Comment:
refactor the db.js database queries to be inside this file and make sure the db.js contains no more database logic